plugin/core/sessions.py

```diff
@@ -349,6 +349,13 @@ def get_initialize_params(variables: Dict[str, str], workspace_folders: List[Wor
         "codeLens": {
             "dynamicRegistration": True
         },
+        "inlayHint": {
+            "dynamicRegistration": True,
+            "resolveSupport": {
+                # not sure if I can just put "command" because the command is nested in the label
+                "properties": ["textEdits", "command"]
```

Comment:
todo, I am not sure of what to put for the command,
`"label.command"` or `"command"`

Reply:
Spec has an example using `label.location` so it should be `label.command` since it's not a root property of the inline hint object.